api/namespace/fund/data_store/store.py

```diff
@@ -27,12 +29,28 @@ def __init__(self):
     def get_funds(self):
         return json.loads(json.dumps(self.funds, default=str))
 
+    @property
+    def applications_index(self):
+        applications = []
+        for fund_name, fund_applications in self.funds.items():
```

Comment:
It will be nice when we can rewrite these loops with more efficient db queries :)

Reply:
Yep